pkgs/servers/web-apps/kavita/default.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  buildNpmPackage,
+  autoPatchelfHook,
+}:
+buildDotnetModule rec {
+  pname = "kavita";
+  version = "0.7.1.4";
+
+  src = fetchFromGitHub {
+    owner = "kareadita";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-jNhiwyz6iVSLlvMNjI689TwQYuEvTJ+QaPvvDQ4UOwc=";
+  };
+
+  patches = [
+    # Disable swagger PostBuild, I couldn't get it to run
+    ./disable-swagger.diff
```

Comment:
`buildDotnetModule` does release build by default, so `swagger` is disabled even without patching, I think.

Reply:
I didn't know that, thanks! I guess I only got the issue while building imperatively and assumed it would be necessary here too.